theatre/studio/src/Studio.ts

```diff
@@ -217,6 +221,30 @@ export class Studio {
     this._store.redo()
   }
 
+  copyKeyframes(keyframes: Keyframe[]) {
+    this._store.copiedKeyframes = keyframes
+    console.info('Keyframes copied:', this.getCopiedKeyframes())
+  }
+
+  getCopiedKeyframes(): Keyframe[] {
+    return this._store.copiedKeyframes
+  }
+
+  pasteKeyframes(leaf: SequenceEditorTree_Row<unknown>, keyframes: Keyframe[]) {
+    if (leaf.sheetObject && leaf.pathToProp) {
+      const pointerToProp = pointerDeep(
+        leaf.sheetObject.propsP,
+        leaf.pathToProp,
+      ) as Pointer<$IntentionalAny>
+
+      this.transaction((api) => {
+        keyframes.forEach(({value}) => {
+          api.set(pointerToProp, value)
```

Comment:
`api.stateEditors.coreByProject.historic.sheetsById.sequence.replaceKeyframes` should be used here, because `api.set()` basically sets the value of a prop at the current sequence position